lib/engine/step/g_1822/track.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require_relative '../base'
+require_relative 'tracker'
+
+module Engine
+  module Step
+    module G1822
+      class Track < Track
+        include Tracker
```

Comment:
what is the point of this class?

Reply:
As it is now its just to include the Tracker module where i override the legal_tile_rotation? method to be able to remove a town from a tile when upgrading to a yellow city.

I still have a few things to do with tracklay and a few methods to override. I have the English chanel, the river crossings cost money when you build into it. So it cost money from both sides not only when you connect.

There a re few private companies that modify the track a well.

But yes as it is now i could have put the method in the tracker.rb in the track.rb class and removed the tracker.rb class.